src/molecules/multiLineInput/component.tsx

```diff
@@ -53,6 +53,7 @@ interface IMultiLineInputProps extends IMoleculeProps<IMultiLineInputTheme> {
 
 export const MultiLineInput = (props: IMultiLineInputProps): React.ReactElement => {
   const [rowCount, setRowCount] = React.useState(props.minRowCount);
+  const maxRowCount = Math.max(props.maxRowCount, props.minRowCount);
```

Comment:
why would max be less than min? this looks like we should raise an error (or at least log a warning) for the user. This line def just looks confusing though cos it looks like the bad behaviour is allowed but it defienitely should not be

Reply:
Yes, but I think if user sets minrow value then if it's greater than default value of maxrow so max should be set equal to min if maxrow is not defined by user.